.github/workflows/ci-pullrequest-main.yml

```
@@ -79,12 +79,44 @@ jobs:
   PackageDeploy:
     runs-on: ubuntu-latest
     needs: BuildTest
+    permissions:
+      contents: read
+      packages: write
+      id-token: write
 
     steps:
     - uses: actions/checkout@v3
     
     - name: Docker Setup BuildX
       uses: docker/setup-buildx-action@v2
+
+    - name: Set image tag
+      run: |
+        echo "DOCKER_TAG_SHA=ghcr.io/buerokratt/mock-classifier:${GITHUB_SHA}" >> $GITHUB_ENV
+        echo "DOCKER_TAG_LATEST=ghcr.io/buerokratt/mock-classifier:latest" >> $GITHUB_ENV
     
     - name: Docker Build
-      run: docker image build src/MockClassifier.Api/
+      run: docker image build src/MockClassifier.Api/ --tag $DOCKER_TAG_SHA
```

```suggestion
      working-directory: src/MockClassifier.Api
      run: docker image build . --tag $DOCKER_TAG_SHA
```
would be a 'nice-to-have' :)